blog-posts/github-billing-dashboard.md

```diff
@@ -0,0 +1,169 @@
+---
+path: "/blog/github-billing-dashboard/"
+date: "2021-07-15"
+title: "GitHub CSV Billing Dashboard"
+featuredImage: images/space.jpg
+author: "Fabian & Felix"
+authorSummary: "Developers at Satellytes"
+attribution:
+creator: upklyak
+source: https://www.freepik.com/free-vector/space-landscape-night_13748451.htm
+---
+
+
+
+> You can try out the dashboard here: [https://satellytes.github.io/github-billing-dashboard](https://satellytes.github.io/github-billing-dashboard)
+
+We use CI/CD pipelines and automatizations in every project we are responsible for. Especially Github Actions is becoming more popular, as it’s simple to use and growing in functionality. Over the last couple of months, we managed to create multiple projects and repositories that are using Github Actions extensively.
+
+Github Actions provides a generous free tier, so costs for smaller projects are often nonexistent. Once you reach the free tier limits, every Action (and other services) needs to be "paid by use" at the end of the month. Github aggregates all costs and shows them in your settings. But Github doesn't provide charts or a dashboard to get into the details of the costs, for example for historical data.
+
+This can be a problem if you have multiple projects, as it’s not obvious which service causes the costs. If you want to figure that out, you can get a CSV from Github, which is hard to read and analyze. We, therefore, decided to create a dashboard to visualize this CSV and show you the costs per repository and month.
+
+![screenshot of the billing settings in a Github account](./images/screenshot-of-gh-billing-settings.png)
+
+This is all the billing information you get from the Github Settings. No historical data, no expense per service.
```

Comment:
The image appears without preparing the reader. I think the description text should come before and then refer to the image with "as in the image below" or similar.

Reply:
Good point 👍  We have now written the description in front of the screenshot. We have also changed the screenshots themselves. In the screenshot, you can now see the menu bar of the browser, which should make the whole thing a bit clearer

Reply:
Yeah much much better. Thank you.